examples/isotp_send.py

```diff
@@ -1,7 +1,6 @@
 #! /usr/bin/python3
```

Comment:
This line only makes sense if the file is an executable script (for example in the `__main__.py` file), otherwise it's just noise